netstandard/pkg/shims/netstandard/System.Security.Cryptography.Cng.Forwards.cs

```diff
@@ -2,6 +2,7 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle))]
+[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.Cryptography.AesCng))]
```

Comment:
Yes. We decided to expand our "in" vs "on" netstandard scope to make sure anything that was targeting netstandard1.6 using any of our packages (either in or on) and retargeted to netstandard2.0 would still compile and work for the scenarios they worked for before (ie run on netcoreapp and later versions of netfx).